### PR TITLE
Add Git Catalog docs

### PR DESCRIPTION
**Pull request by omnomagonz**
Adding the initial Git Catalog documentation to coincide with the release of this new feature.

### CODE REVIEW

**Comment by omnomagonz** [comment]
@rberrelleza I think I did this correctly 😅 

cc @pchico83

**Comment by AgustinRamiroDiaz** [comment]
@huguestennier how can we enable previews for this pr?

**Comment by maroshii** [comment]
I think it's been done from a fork so it's not picking up the auto preview

File: versioned_docs/version-1.2/administration/gitcatalog.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
```
Comment (maroshii) [inline versioned_docs/version-1.2/administration/gitcatalog.mdx:1]: `versioned_docs/{version}` is autogenerated by our framework when we create a new release. I think this need to go in `/src/content/administration`

**Comment by huguestennier** [comment]
> @huguestennier how can we enable previews for this pr?

Since @omnomagonz doesn’t have a Netlify seats, someone with a seat needs to "accept" the deploy preview.

We have a fixed amount of seats (6) and right now it’s Me, @rberrelleza, @RinkiyaKeDad, @jmacelroy, @rlamana and @bigash42.

We ** can ** turn on auto-accepting deploy previews for everyone, but that means anyone that creates a PR will count as a seat and added to the monthly invoice. I’ll let y’all weigh in, but I think we should just have someone accept the deploy preview to keep lower costs

**Comment by pchico83** [comment]
> > @huguestennier how can we enable previews for this pr?
> 
> Since @omnomagonz doesn’t have a Netlify seats, someone with a seat needs to "accept" the deploy preview.
> 
> We have a fixed amount of seats (6) and right now it’s Me, @rberrelleza, @RinkiyaKeDad, @jmacelroy, @rlamana and @bigash42.
> 
> We ** can ** turn on auto-accepting deploy previews for everyone, but that means anyone that creates a PR will count as a seat and added to the monthly invoice. I’ll let y’all weigh in, but I think we should just have someone accept the deploy preview to keep lower costs

@huguestennier I think creating a branch on `okteto/docs` instead of a fork also works

**Comment by rberrelleza** [comment]
closing in favor of https://github.com/okteto/docs/pull/270

**Pull request closed without merging**